jerry-core/ecma/builtin-objects/ecma-builtin-function-prototype.cpp

```diff
@@ -279,10 +279,36 @@ ecma_builtin_function_prototype_object_bind (ecma_value_t this_arg, /**< this ar
      * See also: ecma_object_get_class_name
      */
 
-    /* 17. */
+    /* 16. */
     ecma_number_t *length_p = ecma_alloc_number ();
-    *length_p = 0;
+    *length_p = ECMA_NUMBER_ZERO;
```

Comment:
Could you, please, move the operation to `else` block of `if (ecma_object_get_class_name (this_arg_obj_p) == LIT_MAGIC_STRING_FUNCTION_UL)`?